lib/rules/classic-decorator-hooks.js

```diff
@@ -0,0 +1,62 @@
+let inClassicClass = false;
```

Comment:
I think these should be down inside the `create` method, I’m concerned about these module scope variables being “global state”